.eslintrc

```diff
@@ -115,6 +115,21 @@
       "rules": {
         "oppia/no-testonly": "off"
       }
+    },
+     {
+      "files": ["*.js", "*.ts"],
+      "excludedFiles": [
+      "Polyfills.ts",
+      "ck-editor-copy-content-service.spec.ts",
+      "unit-test-utils.ajs.ts",
+      "mathjax.directive.ts",
+      "math-expression-content-editor.component.ts",
+      "rte-output-display.component.spec.ts",
+      "core/tests/*"
+      ],
+      "rules": {
+        "oppia/disallow-innerHTML": "error"
```

Comment:
I think 'no-inner-html` would be a better name. Also, try to keep all the rule names in lowercase separated by hyphen.

Reply:
Done!